pkg/hostagent/setup.go

```diff
@@ -284,6 +292,21 @@ func (agent *HostAgent) configureContainerIfaces(metadata *md.ContainerMetadata)
 			}
 		}
 
+		for _, ip := range iface.IPs {
+			// We are handling 4 cases  here: ipv4 only, ipv6 only, ipv6 and ipv4, and ipv4 and ipv6
```

Comment:
I don't think this comment applies with the current code

Reply:
My suggestion would be to say:
There are 4 cases: IPv4-only, IPv6-only, dual stack with either IPv4 or IPv6 as the first address. We are guaranteed to derive the MAC address from the  IPv4 address in the IPv4-only case.

Reply:
Okay sounds Good